frontend/src/lib/codemirror/languageServer.ts

```diff
@@ -322,10 +322,6 @@ class LanguageServerPlugin implements PluginValue {
         }
     }
 
-    requestDiagnostics(view: EditorView) {
```

Comment:
I reckon we should keep this, ideally clangd provides live diagnostics in the near future